### ISSUE TITLE
Override response handlers

### ISSUE THREAD

**Issue by cflipse** [rom-rb/rom-http#13]
different HTTP methods are going to have different needs from their response handlers; it should be easy to select a separate response handler to use, so that the same `call` method doesn't have to handle both `DELETE` and `GET` responses.

**Comment by solnic** [comment]
@cflipse is this still needed?

**Comment by AMHOL** [comment]
@solnic I don't think this is needed, I had a similar case when developing a PoC a few years back and just handed off to another class based on the verb in my adapters request/response handlers, this also allows you to do any processing that is common to all requests before handing off to dependencies.
